### PR TITLE
Install gettext 0.21

### PR DESCRIPTION
**Pull request by cedric-anne**

### CODE REVIEW

File: circleci-env-core/Dockerfile [hunk 1]
```diff
@@ -65,6 +65,38 @@ RUN \
...
+  && ln -s /usr/local/gettext/bin/msgfilter /usr/local/bin/ \
+  && ln -s /usr/local/gettext/bin/msgfmt /usr/local/bin/ \
+  && ln -s /usr/local/gettext/bin/msggrep /usr/local/bin/ \
+  && ln -s /usr/local/gettext/bin/msginit /usr/local/bin/ \
+  && ln -s /usr/local/gettext/bin/msgmerge /usr/local/bin/ \
+  && ln -s /usr/local/gettext/bin/msgunfmt /usr/local/bin/ \
+  && ln -s /usr/local/gettext/bin/msguniq /usr/local/bin/ \
+  && ln -s /usr/local/gettext/bin/ngettext /usr/local/bin/ \
+  && ln -s /usr/local/gettext/bin/recode-sr-latin /usr/local/bin/ \
+  && ln -s /usr/local/gettext/bin/xgettext /usr/local/bin/ \
```
Comment (trasher) [inline circleci-env-core/Dockerfile:97]: Maybe more simple to do something like:
```suggestion
  && ln -s /usr/local/gettext/bin/* /usr/local/bin/ \
```

Comment (trasher) [reply circleci-env-core/Dockerfile:97]: Or to add `/usr/local/gettext/bin/` to PATH (dunno if that can be simple in present case)

**Review by trasher — APPROVED** [review]
LGTM anyways (if CI passes ;p)

File: circleci-env-core/Dockerfile (hunk 1 above)
Comment (cedric-anne) [reply circleci-env-core/Dockerfile]: Done.

**Review by orthagh — APPROVED** [review]

**Pull request merged**